src/main/java/cms/gov/madie/measure/services/BundleService.java

```diff
@@ -40,7 +44,28 @@ public ResponseEntity<byte[]> exportBundleMeasure(Measure measure, String access
     if (measure == null) {
       return null;
     }
-
+    if (measure.getMeasureMetaData() != null) {
+      if (measure.getMeasureMetaData().getDevelopers() == null
+          || measure.getMeasureMetaData().getDevelopers().isEmpty()) {
```

Comment:
```suggestion
      if (CollectionUtils.isEmpty(measure.getMeasureMetaData().getDevelopers())) {
```
Assuming getDevelopers() returns a List, you can use either Spring's and Apache's Collection Utils [isEmpty](https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/util/CollectionUtils.html#isEmpty(java.util.Map)) method to combine these two checks.

Reply:
Thank you, TIL